conncheck.go

```diff
@@ -12,17 +12,14 @@
 package mysql
 
 import (
-	"errors"
-	"io"
+	"fmt"
 	"net"
 	"syscall"
-)
 
-var errUnexpectedRead = errors.New("unexpected read from socket")
+	"golang.org/x/sys/unix"
```

Comment:
Does this work on Windows?

Reply:
No, it doesn't, as current implementation too.